ProjectTemplates/templates/ComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -205,11 +205,56 @@
           ]
         },
         {
-          "condition": "(!IndividualLocalAuth || !Hosted)",
+            "condition": "(!IndividualLocalAuth || !Hosted)",
+            "exclude": [
+                "Server/Areas/Identity/Pages/Shared/_LoginPartial.cshtml",
+                "Server/Controllers/OidcConfigurationController.cs",
+                "Server/Models/ApplicationUser.cs"
+            ]
+        },
+         {
+          "condition": "(!GenerateApi)",
+            "exclude": [
+              "Client/Pages/CallWebApi.razor",
+              "Server/Controllers/CallWebApiController.cs"
+            ]
```

Comment:
I think we also want to exclude "Client/Shared/NavMenu.CallsWebApi.razor"

Reply:
note that I pushed a small commit to fix a small indentation issue because GitHub was very confused, and lines were appearing at the bottom, which are at the top